code/modules/projectiles/guns/misc/chem_gun.dm

```diff
@@ -45,3 +106,16 @@
 	if(chambered && !chambered.BB)
 		chambered.newshot()
 	last_synth = world.time
+
+//Smart dart version of the reagent launcher
+/obj/item/gun/chem/smart
+	name = "smartdart repeater rifle"
+	desc = "An experimental improved version of the smart dart rifle. It synthesizes medicinal smart darts which it fills using an inserted hypovial. It can accommodate both large and small hypovials."
```

Comment:
```suggestion
	desc = "An experimental improved version of the smartdart rifle. It synthesizes medicinal smart darts which it fills using an inserted hypovial. It can accommodate both large and small hypovials."
```